src/gleam_stdlib.mjs

```diff
@@ -94,8 +94,14 @@ export function string_length(string) {
 }
 
 function graphemes_iterator(string) {
+  let env_locale = () => {
+    if (typeof navigator.language !== "undefined") {
+      return navigator.language;
+    }
+    return Intl.DateTimeFormat().resolvedOptions().locale;
+  };
   if (Intl && Intl.Segmenter) {
-    return new Intl.Segmenter("en-gb").segment(string)[Symbol.iterator]();
+    return new Intl.Segmenter(env_locale).segment(string)[Symbol.iterator]();
```

Comment:
FWIW I believe you can omit any arguments and get the same result here. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter/Segmenter

```suggestion
  if (Intl && Intl.Segmenter) {
    return new Intl.Segmenter().segment(string)[Symbol.iterator]();
  }
```

Reply:
That's here https://github.com/gleam-lang/stdlib/pull/304